src/collections/_documentation/workflow/apm/index.md

```diff
@@ -44,6 +44,36 @@ We are creating spans for the following operations within a transaction:
 
 #### Managing Transactions
 
-#### Adding Spans
+Let's say you want to create your a transaction for a expensive operation `process_item`
+and send the result to Sentry:
+
+```python
+import sentry_sdk
+
+while True:
+     item = get_from_queue()
+
+     with sentry_sdk.start_span(op="task", transaction=item.get_transaction()):
+         # process_item may create more spans internally (see next examples)
+         process_item(item)
+```
+
+#### Adding additional Spans to the transaction
+
+Consider the next example is somewhere called in the `process_item` function from before.
```

Comment:
Corrections below:
"Consider the next example is called somewhere in the `process_item` function from before. Our SDK can determine if there is a current open `transaction` and add all newly created spans as child operations to the transaction. Keep in mind; each individual span also needs to be finished; otherwise, it will not show up in the `transaction`."